lib/ReactViews/Map/MapNavigation.jsx

```diff
@@ -141,6 +141,7 @@ class MapNavigation extends React.Component {
               <If
                 condition={
                   !this.props.terria.configParameters.disablePedestrianMode &&
+                  !this.props.viewState.useSmallScreenInterface &&
```

Comment:
this may cause a problem if a user starts pedestrian mode and then resizes their screen. Perhaps another check to see whether they are already in pedestrian mode?

Reply:
Good point! If a `ToolButton` unmounts it [closes the tool automatically](https://github.com/TerriaJS/terriajs/blob/6296fccd92983b5f0154dd47061dd438b7941ed1/lib/ReactViews/Tool.tsx#L85) so the pedestrian mode will exit correctly when switching to mobile view.